app/assets/stylesheets/notifications.scss

```diff
@@ -428,3 +431,134 @@
     @include load-more;
   }
 }
+
+
+.notifications-filter-menu__mobile {
+  display: none;
+
+  @media screen and (max-width: $breakpoint-m) {
+    display: block;
+  }
+}
+
+// Notifications filtering on mobile.
+.notifications-mobile-filters {
+  @media screen and (min-width: 950px) {
+    display: none;
+  }
+
+  display: none;
+
+  &.showing {
+    display: flex;
+  }
+
+  background:  var(--base-inverted);
+}
+
+.crayons-dropdown {
```

Comment:
`crayons-dropdown` is already existing component in Crayons (https://dev.to/page/crayons - cmd+f for "dropdown" since I can't link it :D). It's the one we use for profile dropdown in header for example (and few other places too).

But in the markup you don't actually use the `crayons-dropdown` itself - instead it's `crayons-dropdown__*` without this parent element.

Also, since dropdowns already exits in Crayons, any thoughts about reusing them instead of introducing new one? I'm not talking code-wise but also design-wise.

Reply:
I think this is something we can iterate on a little bit. I think that the existing `crayons-dropdown` menu which uses the box styling works really great on desktop, but as we scale down to mobile, I want to design out a version that felt mobile-first. I think this is something that would take more consideration if we're thinking about it for our crayons design system, so what I can do in this file is rename it so that it's specific to Notifications' filtering menu, and revisit this at a later time when we've worked out those details together.

Reply:
Sounds good and I definitely agree about renaming this one and coming up with better mobile version for actual `crayons-dropdown`!